tests/nimble_statconn_gnrc_ext/Makefile

```diff
@@ -0,0 +1,23 @@
+BOARD ?= nrf52dk
+include ../Makefile.tests_common
+
+# include shell support
+USEMODULE += shell
+USEMODULE += shell_commands
+USEMODULE += ps
+
+# Include GNRC and RPL
+USEMODULE += auto_init_gnrc_netif
+USEMODULE += gnrc_ipv6_router_default
+USEMODULE += auto_init_gnrc_rpl
+USEMODULE += gnrc_rpl
+USEMODULE += gnrc_icmpv6_echo
```

Comment:
Is rpl needed in this example?

Reply:
oh wait, no, not strictly, but then again it is useful while testing so users do not have to configure static routes but may opt to use RPL instead...

Reply:
Ok!